pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/impl/stats/BytesColumnPredIndexStatsCollector.java

```diff
@@ -59,11 +59,11 @@ public BytesColumnPredIndexStatsCollector(String column, StatsCollectorConfig st
   private void collectEntry(Object entry, ObjectSet<ByteArray> set) {
     if (entry instanceof Object[]) {
       for (final Object e : (Object[]) entry) {
-        byte[] value = (byte[]) e;
-        set.add(new ByteArray(value));
+        ByteArray value = ((ByteArray) e);
```

Comment:
We don't support MV for BYTES. So I suggest throw UnsupportedOperationException here or add a TODO.
(OCD) Remove redundant parentheses.